src/library/VegetablesCard/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './styles.css';
+
+function CardItems({ results }) {
+    return results.map(({ data, id }) => (
+        <div key={id} className="prismic-cms-vegetables-card">
+            <div className="prismic-cms-vegetables-card_img-container">
+                <img className="prismic-cms-vegetables-card_img" src={data.image.url} alt="" />
+            </div>
+            <div className="prismic-cms-vegetables-card_description-container">
+                <h1 className="prismic-cms-vegetables-card_header">{data.name[0].text}</h1>
+                <p className="prismic-cms-vegetables-card_description">{data.description[0].text}</p>
+            </div>
+        </div>
+    ));
+}
+
+export default function VegetablesCard(props) {
```

Comment:
`VegetableCard` in singular

Reply:
I see this component is the container, well then `ExampleCard` since not all examples are going to be vegetables